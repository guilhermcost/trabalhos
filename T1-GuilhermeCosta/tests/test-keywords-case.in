if else
If Else
int Int
return Return
void Void
while While
for FOR
Const const

