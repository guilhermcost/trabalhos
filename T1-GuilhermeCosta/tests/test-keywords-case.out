(1,KEY,"if")
(1,KEY,"else")
(2,ID,"If")
(2,ID,"Else")
(3,KEY,"int")
(3,ID,"Int")
(4,KEY,"return")
(4,ID,"Return")
(5,KEY,"void")
(5,ID,"Void")
(6,KEY,"while")
(6,ID,"While")
(7,KEY,"for")
(7,ID,"FOR")
(8,ID,"Const")
(8,KEY,"const")
